plugins/source/gandi/codegen/recipes/livedns.go

```diff
@@ -0,0 +1,43 @@
+package recipes
+
+import (
+	"github.com/cloudquery/plugin-sdk/codegen"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/go-gandi/go-gandi/livedns"
+)
+
+func LiveDNSResources() []Resource {
+	return []Resource{
+		{
+			DataStruct:   &livedns.Domain{},
+			PKColumns:    []string{"fqdn"},
+			ExtraColumns: []codegen.ColumnDefinition{SharingIDColumn},
+			Relations:    []string{"DomainSnapshots()"},
+			TableName:    "gandi_livedns_domains",
+
+			Template:         "resource_manual",
+			Package:          "livedns",
+			TableFuncName:    "Domains",
+			Filename:         "domains.go",
+			ResolverFuncName: "fetchDomains",
```

Comment:
I think most of these fields should ideally be inferred from a single entry and changed to the right case using `caser`, like `SubService` in the aws plugin case.

Reply:
I agree these were copied from the cloudflare plugin with minor modifications, when we have a more generic/basic codegen for small plugins we can replace these in all plugins.

Reply:
Fixed